tests/storage-vm

```diff
@@ -484,6 +484,32 @@ for poolDriver in $poolDriverList; do
 
         lxc delete -f v1
 
+        echo "==> Checking setting instances.migration.stateful for VMs"
+        # Check that migration.stateful is set in the instance's expanded config to the value of `instances.migration.stateful`
+        # if no local instance config nor profile value of `migration.stateful` is set.
+
+        lxc config set instances.migration.stateful=true
+        lxc init "${TEST_IMG:-ubuntu-minimal-daily:22.04}" v1 --vm -s "${poolName}"
+
+        lxc query /1.0/instances/v1 | jq '.expanded_config["migration.stateful"]' | grep -q true
```

Comment:
```suggestion
        lxc query /1.0/instances/v1 | jq -r '.expanded_config["migration.stateful"]' | grep -xF true
```
I would prefer to use `jq` `-r` to avoid the quoting and then use a full line pattern matching with `grep`. No silencing `grep`'s output is also useful when debugging using log output.

Reply:
fixed